common/live/stg/infra/dev/tyler/terraform.tfvars

```diff
@@ -0,0 +1,18 @@
+# ↓ Module metadata
+
+terragrunt = {
+  terraform {
+    source = "/project/modules//gcp-project"
+  }
+  dependencies {
+    paths = ["../zone"]
+  }
+  include = {
+    path = "${find_in_parent_folders()}"
+  }
+}
+
+# ↓ Module configuration (empty means all default)
+
+project_name    = "dev-tyler"
+project_owner   = "user:tyler@raisingthefloor.org"
```

Comment:
I guess that this is for debugging/develop the solution.

Reply:
I thought I wanted a permanent project in the test org, but in fact I don't. I reverted this.